docs/src/administration/users.md

```diff
@@ -1,138 +1,162 @@
+### Change user permissions from the console
 
-For example, if you invite a user with **Project Admin** role to a project on Platform.sh, but you haven't invited them to the remote repository on GitHub / GitLab, they might be [unable to clone the project](/administration/web/_index.md#git) locally.
+To change the user permissions, follow the steps:
 
-In this example, using either `platform get` with the CLI:
+{{< codetabs >}}
+---
+title="In the console"
+---
+1. Go to your console and select the project where you want to change the user permissions.
+2. Under **Settings**, click **Access**.
+3. Select a user and change the permissions.
+4. Click **Save**.
+5. After you delete a user from a project or an environment type, you must trigger a redeploy to propagate SSH access changes to each environment. You can redeploy by clicking **Redeploy** in the console.
+---
+title="Using the CLI"
+---
+Use the [Platform.sh CLI (Command Line Interface)](/development/cli/_index.md)to change user permissions.
 
-```bash
-$ platform get <projectID>
+* `platform user:role`
 
-```
+  ```bash
+  platform user:role user1@example.com -r contributor
+  ```
 
-or the `git clone` command visible from the "Git" dropdown in the management console
+Use `platform list` to get the full list of commands.
 
-```bash
-$ git clone git@github.com:user/github-repo.git Project Name
-```
+After you delete a user from a project or an environment type, you must trigger a redeploy to propagate SSH access changes to each environment. You can redeploy by using the CLI command platform `redeploy`.
 
-both would error with
+{{< /codetabs >}}
 
-```bash
-Failed to connect to the Git repository: git@github.com:user/github-repo.git
 
-Please make sure you have the correct access rights and the repository exists.
-```
+### Transfer a project ownership
 
-despite their **Project Admin** access to the project.
+You can transfer your plan ownership to a different [organization](https://docs.platform.sh/administration/organizations.html) anytime.
 
-This enhaces consistency over your source code, as the Platform.sh project functions as a read-only mirror of your remote repository. Otherwise, changes pushed directly to the project would be overwritten or deleted when commits are pushed via the integration. Platform.sh considers your integrated remote repository to be the "source of truth" as soon as it has been configured, and this caveat ensures that all commits go through the integration.
+#### Before you begin
 
-The best course of action is to have your access updated on the integrated repository. If for some reason that is not a quick change, you can still clone through the project using the legacy pattern (which will set the *project* as its remote), but again, it is not recommended that you commit to the project once you have done so:
+You have to be the organization owner or an organization user with the **manage plans** permission.
```

Comment:
As I noted above, I'm not sure if the permission is correct but it should at least link to a definition of that permission.

Reply:
As above, it should have both permissions and with links (they might not read above, only here).